applications/fishing-map/src/features/report/ReportFilter.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import cx from 'classnames'
+import { TagList } from '@globalfishingwatch/ui-components'
+import { TagItem } from '@globalfishingwatch/ui-components/dist/tag-list'
+import { UrlDataviewInstance } from 'types'
+import styles from 'features/workspace/shared/LayerPanel.module.css'
+
+type ReportFilterProps = {
+  label: UrlDataviewInstance
```

Comment:
This has to be a typo, I've replaced by `string` in order to be able to run the project